api/internal/store/service/schedule_test.go

```diff
@@ -0,0 +1,154 @@
+package service
+
+import (
+	"context"
+	"testing"
+
+	"github.com/and-period/furumaru/api/internal/exception"
+	"github.com/and-period/furumaru/api/internal/store"
+	sentity "github.com/and-period/furumaru/api/internal/store/entity"
+	"github.com/and-period/furumaru/api/internal/user"
+	uentity "github.com/and-period/furumaru/api/internal/user/entity"
+	"github.com/and-period/furumaru/api/pkg/jst"
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCreateSchedule(t *testing.T) {
+	t.Parallel()
+
+	producersIn := &user.MultiGetProducersInput{
+		ProducerIDs: []string{"producer-id01", "producer-id02"},
+	}
+	producers := uentity.Producers{
+		{
+			AdminID: "producer-id01",
+		},
+		{
+			AdminID: "producer-id02",
+		},
+	}
+	productsIn := &store.MultiGetProductsInput{
+		ProductIDs: []string{"product-id01"},
+	}
+	products := sentity.Products{
+		{
+			ID: "product-id01",
+		},
+	}
+
+	tests := []struct {
+		name      string
+		setup     func(ctx context.Context, mocks *mocks)
+		input     *store.CreateScheduleInput
+		expect    *sentity.Schedule
+		expectErr error
+	}{
+		{
+			name: "success",
+			setup: func(ctx context.Context, mocks *mocks) {
+				mocks.user.EXPECT().MultiGetProducers(gomock.Any(), producersIn).Return(producers, nil)
+				mocks.db.Product.EXPECT().MultiGet(gomock.Any(), productsIn).Return(products, nil)
```

Comment:
```suggestion
				mocks.db.Product.EXPECT().MultiGet(gomock.Any(), []string{"product-id01"}).Return(products, nil)
```

Reply:
https://github.com/and-period/furumaru/blob/19e0a4942331410f393a1c2937e9bec8893c9df4/api/internal/store/database/database.go#L57 のInterface定義みてると↑の感じかも👀

Reply:
```
Unexpected call to *mock_database.MockProduct.MultiGet([context.Background.WithCancel.WithCancel [product-id01]]) at /Users/kaito/Documents/calmato/marche/api/internal/store/service/schedule.go:36 because:
        expected call at /Users/kaito/Documents/calmato/marche/api/internal/store/service/schedule_test.go:48 has already been called the max number of times
```
なんか呼び出し回数が限界超えた…？